pages/src/navigation/sidebar/sidebar-share.less

```diff
@@ -0,0 +1,62 @@
+@import (reference) '../../common/variables.less';
+
+esl-d-sidebar {
+  .sidebar-nav-item-share {
+    display: flex;
+    align-items: center;
+
+    visibility: visible;
+    opacity: 1;
+    transition:
+      visibility 0s,
+      opacity 0.25s;
+
+    padding: 15px 40px 15px 25px;
+    @media @sm-xl {
+      margin-top: auto;
+      margin-bottom: 0;
+      padding: 20px 25px 0;
```

Comment:
```suggestion
      padding: 20px 21px 0;
```

mb make it like `padding-left` for **sidebar-nav-item-heading**? then it will go into one line and will be to the left

<img width="1174" alt="Screenshot 2023-07-24 at 13 30 07" src="https://github.com/exadel-inc/esl/assets/52083788/2fc4e5aa-ec3b-42b9-923c-a60dadcac6f9">